pkg/psmdb/pbm/exec.go

```diff
@@ -0,0 +1,16 @@
+package pbm
+
+import (
+	"context"
+	"io"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/percona/percona-server-mongodb-operator/clientcmd"
+)
+
+const BackupAgentContainerName = "backup-agent"
+
+func exec(ctx context.Context, cli *clientcmd.Client, pod *corev1.Pod, container string, command []string, stdin io.Reader, stdout, stderr io.Writer) error {
```

Comment:
WDYT about having `exec` that declares these `io.Writer`s, that way you don't repeat these `stdout` and `stderr` all over the place. But it can accept like generic `any` type param to which you can unmarshal the result.
So you could have something like

```
var response  BackupResponse{}
err = exec(ctx, cli, pod, BackupAgentContainerName, cmd, &response)

// or where the response is not needed.
err = exec(ctx, cli, pod, BackupAgentContainerName, cmd, nil)

```

Reply:
Also I can see that you always pass this `BackupAgentContainerName` as the arg for `container` param, you can probably remove it.